publications/management/commands/load_global_regions.py

```diff
@@ -0,0 +1,88 @@
+import os
+import io
+import zipfile
+import urllib.request
+from django.conf import settings
+from publications.models import GlobalRegion
+from django.core.management.base import BaseCommand
+from django.contrib.gis.gdal import DataSource
+
+COMMAND_DIR = os.path.dirname(__file__)
+CONTINENTS_URL     = (
+    "https://services.arcgis.com/P3ePLMYs2RVChkJx/arcgis/rest/services/"
+    "World_Continents/FeatureServer/0/query"
+    "?where=1%3D1&outFields=*&f=geojson"
+) # Esri Hub World Continents Feature Service 
+CONTINENTS_FILE    = "world_continents.geojson"
+
+OCEANS_WFS_URL     = (
+    "https://geo.vliz.be/geoserver/MarineRegions/wfs?"
+    "service=WFS&version=1.0.0&request=GetFeature&"
+    "typeName=MarineRegions:iho&outputFormat=application/json"
+) # MarineRegions IHO Sea Areas WFS
+OCEANS_FILE        = "marine_regions_iho.geojson"
+
+class Command(BaseCommand):
+    help = "Load 7 continents (Esri Hub) and 5 oceans (MarineRegions IHO) into GlobalRegion"
+
+    def handle(self, *args, **options):
+        self.stdout.write("Downloading Esri World Continents…")
+        req = urllib.request.Request(CONTINENTS_URL, headers={"User-Agent":"Mozilla/5.0"})
+        continents_path = os.path.join(COMMAND_DIR, CONTINENTS_FILE)
+        with urllib.request.urlopen(req) as resp, open(continents_path, "wb") as out:
+            out.write(resp.read())
```

Comment:
This will read the whole response to memory and then save it.

I don't know how big the files are, but streaming the file is probably a good idea, see https://docs.python.org/3/library/shutil.html#shutil.copyfileobj for a suitable function:

See example: https://stackoverflow.com/a/48691447